src/mlpack/methods/linear_regression/linear_regression_main.cpp

```diff
@@ -163,7 +163,6 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timer)
       timer.Start("load_responses");
       responses = params.Get<rowvec>("training_responses");
       timer.Stop("load_responses");
-
```

Comment:
Sorry, what I meant was that there was no need to replace the blank line with a line that has four spaces.  I didn't mean that we should remove the line. :+1: